.travis/linux_runner.sh

```diff
@@ -90,10 +103,30 @@ script() {
     ./bin/apisix start
     mkdir -p logs
     sleep 1
+
+    #test grpc proxy
```

Comment:
How about putting the script `test grpc proxy` section into a separate file? This will make this main file simpler.